documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/generating-necessary-inputs.rst

```diff
@@ -0,0 +1,84 @@
+.. _generating-necessary-inputs:
+
+Generate the necessary input files
+#############################################################
+
+The ``sct_compute_compression`` command uses the shape metric of the spinal cord at the level of compression and computes a ratio with non-compressed levels above and below the compression site.
+
+
+To use ``sct_compute_compression``, we need the 3 following input files:
+
+
+1. Spinal cord segmentation
+2. Vertebral labels
+3. Compression labels (``t2_compressed_label-compression.nii.gz``)
+
+You can also get these files by downloading :sct_tutorial_data:`data_normalizing-morphometrics-compression.zip`.
+
+1. Spinal cord segmentation
+----------------------------
+.. code:: sh
+
+   sct_deepseg_sc -i t2_compressed.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+
+:Output files/folders:
+   - ``t2_compressed_seg.nii.gz`` : Spinal cord segmentation.
+
+
+2. Label the spinal cord vertebral levels
+------------------------------------------
+.. code:: sh
+
+   sct_label_vertebrae -i t2_compressed.nii.gz -s t2_compressed_seg.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : The input image file.
+   - ``-s`` : Spinal cord segmentation.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+:Output files/folders:
+   - ``t2_compressed_seg_labeled.nii.gz`` : Spinal cord vertebral labels.
+
+3. Generate spinal cord compression labels
+-------------------------------------------
+
+1. Open the image in FSLeyes. If the image does not have sufficient axial and sagittal resolution, open two individual images (axial image and sagittal image) in FSLeyes.
+
+.. code:: sh
+
+   fsleyes t2_compressed.nii.gz &
+
+2. Look at the clinical data provided with compression information to know at which level the compression(s) are located.
```

Comment:
I'm also confused about what "clinical data" means in this context, since we don't provide any example clinical data in our tutorial dataset. 

Is this something that our users will 100% have? Will this step be confusing to our users if they don't have clinical data?

Reply:
Agree, this might be confusing. By clinical data, we mean information from clinicians about the level of compression(s), for example: 

|  participant_id | compression level  |
|---|---|
|  sub-001 | C3/C4  |
|  sub-002 | C6/C7  |

Maybe we can rephrase this:

```
2. If clinical data with information about compression level(s) is available, look at them to know at which level the compression(s) is located.
```

Reply:
Solved in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4162/commits/144599c93f400758eb939b6c3b25282484d74d27